package.json

```diff
@@ -8,37 +8,42 @@
     "type": "git",
     "url": "github:philips-software/odin"
   },
+  "type": "module",
   "main": "lib/index.js",
   "files": [
     "lib"
   ],
   "scripts": {
-    "build": "tsc --build",
+    "build": "tsc --build tsconfig-refs.json",
     "clean": "rimraf --glob lib **/.eslintcache **/*.log **/*.tsbuildinfo **/coverage",
     "lint": "eslint . --ext .js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --cache --max-warnings 0",
     "release": "semantic-release",
     "test": "vitest run --coverage",
     "test:dev": "vitest dev --coverage",
     "test:ui": "vitest dev --coverage --ui"
   },
+  "dependencies": {
+    "debug": "^4.3.4"
+  },
   "devDependencies": {
     "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
-    "@typescript-eslint/eslint-plugin": "5.60.0",
-    "@typescript-eslint/parser": "5.60.0",
-    "@vitest/coverage-v8": "0.32.2",
-    "@vitest/ui": "0.32.2",
-    "esbuild": "0.18.7",
-    "esbuild-decorators": "1.0.0",
-    "eslint": "8.43.0",
+    "@types/debug": "^4.1.8",
```

Comment:
pin dependency;

Reply:
open source packages should not pin deps, it hinders the users ability to get security fixes